package/MDAnalysis/topology/PDBParser.py

```diff
@@ -303,6 +308,22 @@ def _parseatoms(self):
         masses = guess_masses(atomtypes)
         attrs.append(Masses(masses, guessed=True))
 
+        # Guessed attributes
+        # Need to pull elements from Atom names
+        # Similar to the check for atomtypes function
+        if not any(elements):
+            # Can further add a check_cg_atom(elements) function in guessers.py to check for CG atom.
+            elements = guess_atom_element(elements)
+            if elements == '':
```

Comment:
Isn't `elements` a list?

Reply:
Yeah I think you'll want something like `any(not e for e in elements)` to detect missing elements